### PR TITLE
12.1 Release (Full Documentation Update)

### PR DESCRIPTION
**Pull request by peterhallerREL**

### CODE REVIEW

File: docs/trace_connectors_user_guide.md [hunk 1]
```diff
@@ -85,6 +85,9 @@ Refer to the [Merge 1 User Guide](https://s3.amazonaws.com/Merge1Public/User%20G
 4. Globanet pushes the data to Relativity Fileshare's `{FILESHARE_WORKSPACE_ROOT}\DataTransfer\Import\Globanet_Data\{DATA_SOURCE_ARTIFACT_ID}\Drop` folder
 5. Globanet pushes any logs to Relativity Fileshare's `{FILESHARE_WORKSPACE_ROOT}\DataTransfer\Import\Globanet_Data\{DATA_SOURCE_ARTIFACT_ID}\Logs` folder
 
+## Setting up Trace Shipper Service
+If VPN/direct option for data delivery is not available you can configure Trace Shipper Service.  Refer to the [Trace Shipper Service Documentation](Trace Shipper Service.md) for instructions on how to install and configure Trace Shipper Service. Contact support@relativity.com if you need assitance.
```
Comment (peterhallerREL) [inline docs/trace_connectors_user_guide.md]: @brianheinz It seems like we are calling out Shipper as a backup option to VPN/direct option. Is that correct? I'm guessing the priority is Direct, Shipper, then VPN.

File: docs/user_documentation.md [hunk 2]
```diff
@@ -1434,7 +1433,7 @@ Trace automatically extracts metadata information for Microsoft Office 365 Data
...
 | Calculated               | File Extension                | Fixed-Length Text | The extension of the file, as assigned by the processing engine after it reads the header information from the original file |
 | Calculated               | File Name                     | Fixed-Length Text | The original name of the file                                |
-| DEPRECATED / DO NOT USE  | ~~File Size~~                 | Decimal           | This field is automatically provided by Relativity on import. **DO NOT MAP** this field in integration point profile. This field will be removed from Trace Document Extraction Field in the future release of Trace |
+| Calculated               | File Size                     | Decimal           | This field is automatically provided by Relativity on import. However, this field can also be calculated and set by Trace. Trace calculates this field as the size of the native file. **IMPORTANT:** Since both Relativity and Trace can provide a value for this field, it is up to user to decide how to use this field and decide which value to store.                                                      |
```
Comment (peterhallerREL) [inline docs/user_documentation.md]: As a user, I'm confused by what action I need to take here. How do I decide whether Relativity populates it or Trace populates it? What is the difference between Relativity and Trace? What is the difference between the two values?

File: docs/trace_connectors_user_guide.md (hunk 1 above)
Comment (peterhallerREL) [reply docs/trace_connectors_user_guide.md]: Spoke with Misha. This makes sense since Shipper is in BETA.

File: docs/user_documentation.md (hunk 2 above)
Comment (peterhallerREL) [reply docs/user_documentation.md]: @wfreltrace I spoke with Misha on this one. We want to take stronger stance on the action our users should take. Can we go with "**DO NOT MAP** this field in integration point profile. A value for this field is automatically calculated by Relativity on import."

**Review by mishakogan — APPROVED** [review]

File: docs/user_documentation.md (hunk 2 above)
Comment (wfreltrace) [reply docs/user_documentation.md]: @peterhallerREL So change the description to what you've written but leave the "calculated" and "file size" as is in this change?

**Review by mishakogan — APPROVED** [review]

**Pull request merged**